src/components/dropdown/_dropdown.scss

```diff
@@ -256,9 +256,13 @@
     }
   }
 
-  .#{$prefix}--dropdown[data-invalid] {
+  .#{$prefix}--dropdown--invalid {
```

Comment:
Does this change cover React dropdown (`<DropdownV2>`) as well as vanilla dropdown?

Reply:
from what I can tell, the React dropdown does not account for invalid state, so this change won't affect it (and the React component needs to be extended...we can create a new ticket in the React repo for this). please let me know if this is incorrect

Reply:
Oh sorry I had a false assumption that `--invalid` is for React. Makes sense now.

Reply:
Could you update `migrate-to-10.x.md` mentioning the change in how to apply the invalid style? Thanks!

Reply:
sure I updated that document